pages/releases.js

```diff
@@ -37,7 +38,7 @@ const Releases = ({ releases, sidebarPages }) => (
         <Anchor id={release.name}>
           <ReleaseName>{release.name} <Date>{getFormattedDate(release.created_at)}</Date></ReleaseName>
         </Anchor>
-        {md(release.body)}
+        {md(normalizeMdHeadings(release.body, 3))}
```

Comment:
Instead of adding `normalizeMdHeadings` you can just use commonmark's level detection. It's already built in :wink: https://github.com/halvves/styled-components-website/blob/d694dcf1bf9f7dfdf6205be1e220674090ce8d61/components/md.js#L104-L123

There you can more easily add some state for the levels and add a modifier

Reply:
I had initially looked at building it into the `md` function/commonmark handling, but couldn't wrap my mind around how I would detect the level of the first heading in a document and normalize based on that:

so if base level was set to 3 turn `<h1/><h2/>` into `<h3/><h4/>` _and_ turn `<h4/><h5/>` into `<h3/><h4/>` etc...

Do you have thoughts or direction on how I could approach handling that with commonmark's detection (I can take a look again this eve, but wanted to see if you could help freshen my perspective before then)?

Reply:
@philpl - actually it just occurred to me how to go about it... closing this PR and opening a new one.